lib/reporting/reports/enterprise_fee_summary/enterprise_fees_with_tax_report_by_producer.rb

```diff
@@ -1,17 +1,15 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
I would have added this to the todo file because I would like to resolve this problem one day.

Reply:
Sorry, I meant the `.rubocop_todo.ym` file.